clustered/server/src/main/java/org/ehcache/clustered/server/state/EhcacheStateService.java

```diff
@@ -67,4 +68,5 @@
 
   void loadExisting(ServerSideConfiguration configuration);
 
+  EhcacheStateContext beginProcessing(EhcacheMessageType messageType, String name);
```

Comment:
If we're beginning processing... why don't we pass the whole message?  That would read much more naturally, right?